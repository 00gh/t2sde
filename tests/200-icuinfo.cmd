#!/bin/sh

[ "$QEMU" ] || exit 43

cd $SYSROOT
exe=usr/bin/icuinfo
[ -e $exe ] || exit 42

$QEMU -chroot $SYSROOT $exe -v | grep return
