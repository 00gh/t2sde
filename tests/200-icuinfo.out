Plugins are disabled.
ICU Initialization returned: U_ZERO_ERROR
